lib/alerts_viewer_web/live/delay_alert_algorithm/median_instantaneous_minus_scheduled_headway_component .ex

```diff
@@ -0,0 +1,36 @@
+defmodule AlertsViewer.DelayAlertAlgorithm.MedianInstantaneousMinusScheduledHeadwayComponent do
```

Comment:
nitpick: Is there a space before the `.ex` in this filename?

Reply:
Thank you for spotting that, it's very hard to see in the VSCode file browser.